Mage/src/main/java/mage/MageObjectReference.java

```diff
@@ -100,6 +100,9 @@ public MageObjectReference(UUID sourceId, Game game) {
         }
     }
 
+    public String toString(){
```

Comment:
Miss override. Values will be visible by IDE hints while debugging. Also it must show standard object id from 0 to 3, not 0 to 8.

Reply:
Added override and shortened ID to 3 characters.

Yeah, I added this because I was trying to use print-based debugging. I find it easier to look at a wide range of time when looking at a single big text log.